app/installer/.babelrc

```diff
@@ -0,0 +1,40 @@
+{
+    "env": {
+        "development": {
+            "presets": [
+                "es2015",
+                "react",
+                "react-hmre",
+                "stage-0"
```

Comment:
I don't think you need stage-0, as you do not use do expressions neither `::func` notation